c10/core/DispatchKey.h

```diff
@@ -107,6 +108,8 @@ enum class DispatchKey : uint8_t {
   PrivateUse2,
   PrivateUse3,
 
+  NumBackendKeys, // Sentinel, end of backend keys.
```

Comment:
The enum here uses up space in the runtime dispatch table; now you will always have an (unnecessary) gap between PrivateUse3 and Meta.  This probably doesn't matter too much (since PrivateUse1 is already a bunch of extra space) but it would be nice to mention it in case we need to penny pinch space later. Note that you can't conveniently say `NumBackendKeys = PrivateUse3 + 1` as that will cause `Meta` to get offset.